### PR TITLE
feat: add bankInstructions to V3 profile route

### PR DESCRIPTION
**Pull request by eweise**

### CODE REVIEW

**Comment by eweise** [comment]
@olmstead @abuecker

File: server/v3.js [hunk 1]
```diff
@@ -77,6 +77,49 @@ export const v3 = [
           a.description = formats.accountDescription();
         });
 
+        mock.accounts[0].bankInstructions = [
```
Comment (abuecker) [inline server/v3.js]: We should put these inside a `forEach()`, otherwise we'll only get bank instructions for up to the first 2 mocked accounts.

Comment (eweise) [inline server/v3.js]: Seems ok though since not all accounts will have bank instructions and we need to test for that as well on the front end

Comment (abuecker) [inline server/v3.js]: Do we need to coerce the bank instructions at all?  The built in mocker functionality will deliver between 0 and 5 items in an array.

Comment (eweise) [inline server/v3.js]: didn't see them being generated originally but see them now. removed

**Comment by abuecker** [comment]
👍

**Pull request merged**